shared/js/initApp.js

```diff
@@ -1,3 +1,6 @@
+// Attempt to save projects every 30 seconds
```

Comment:
Ah, I see now, the code below is gated above in the method.

`initApp()` is getting (maybe always has been) pretty unwield-ily long, we should keep an eye out for opportunities to extract some named methods out of  it to make it a bit easier to read next time we're in there